### ISSUE TITLE
Docs pdf link broken

### ISSUE THREAD

**Issue by SuperSat001** [numpy/numpy.org#643]
On https://numpy.org/doc/stable/, the **Download Documentation PDF Version** link https://numpy.org/doc/stable/numpy-user.pdf is broken.

Currently it gives a 404.

**Comment by mattip** [comment]
We no longer produce PDF documentation.

**Comment by SuperSat001** [comment]
Then this link should be removed https://i.imgur.com/Dd6eo1U.png

**Comment by mattip** [comment]
Agreed. It will be "fixed automatically" when 1.25 is released because the [devdocs](https://numpy.org/devdocs/index.html) will become the stable docs, and that link was removed there. If someone wants to try their hand at fixing this for 1.24, this is the source for that page https://github.com/numpy/doc/blob/main/1.24/index.html. The link for older versions (which still have pdf documentation) is also broken, it refers to `stable` rather than the relative version.

**Comment by mattip** [comment]
In addition to the [main page of older versions](https://numpy.org/doc/1.24/index.html) having a link to the PDF from stable, which doesn't exist,  https://numpy.org/doc/ has a typo: the NumPy 1.24 pdf download links to the 1.23 pdf file.

**Comment by partev** [comment]
> In addition to the [main page of older versions](https://numpy.org/doc/1.24/index.html) having a link to the PDF from stable, which doesn't exist, https://numpy.org/doc/ has a typo: the NumPy 1.24 pdf download links to the 1.23 pdf file.

this was done intentionally, because 1.24 no longer produced PDF documentation. this is mentioned in [issue 616](https://github.com/numpy/numpy.org/issues/616). I agree that it should probably be removed.
